File: herejong-front/herejong-app/src/app/model/star-form.ts
export class StarForm {

    id: string;
    productId: string;
    starProductPoint: number;
     driverId: string;
    starDriverPoint: number;

    constructor(
  
        productId: string,
        starProductPoint: number,
        driverId: string,
        starDriverPoint: number,

    ) {

        this.productId = productId;
        this.starProductPoint = starProductPoint;
        this.driverId = driverId;
        this.starDriverPoint = starDriverPoint;
     
    }
}

File: herejong-back/herejong/src/main/java/com/here/herejong/message/request/StarForm.java
package com.here.herejong.message.request;



public class StarForm {


    private String  id ;
    private String driverId ;
    private Integer starDriverPoint ;
    private String productId ;
    private Integer starProductPoint ;
    

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDriverId() {
        return driverId;
    }

    public void setDriverId(String driverId) {
        this.driverId = driverId;
    }

    public Integer getStarDriverPoint() {
        return starDriverPoint;
    }

    public void setStarDriverPoint(Integer starDriverPoint) {
        this.starDriverPoint = starDriverPoint;
    }

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId;
    }

    public Integer getStarProductPoint() {
        return starProductPoint;
    }

    public void setStarProductPoint(Integer starProductPoint) {
        this.starProductPoint = starProductPoint;
    }



    
}